test/integration/cart_integration_test.rb

```diff
@@ -3,17 +3,19 @@
 class CartIntegrationTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
 
-  test "a cart starts empty" do
-    visit "/"
+  test 'a cart starts empty' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.